CONTRIBUTING.md

```diff
@@ -28,14 +27,6 @@ We require all commits to be [signed](https://docs.github.com/en/authentication/
 
 If your `user.name` and `user.email` are configured in your Git config, you can sign your commit automatically with `git commit -s`.
 
-## Support Channels
-
-Before opening a new issue or submitting a Pull Request, make sure to search through the [docs](https://github.com/open-component-model/ocm-spec), open and closed issues, open and merged Pull Requests, and the [Discussions](https://github.com/open-component-model/ocm/discussions) board to check whether your question has been raised or answered already.
-
-Please open an issue in any of the repositories in the `open-component-model` organization if you wish to [request a new feature](https://github.com/open-component-model/ocm/issues/new?assignees=&labels=kind%2Fenhancement&template=enhancement_request.md) or [report a bug](https://github.com/open-component-model/ocm/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.md).
-
-If you wish to propose or discuss a more involved feature or change to any of the OCM projects, you could start a new thread in the [`ocm` Discussion Board](https://github.com/open-component-model/ocm/discussions). For example, this could be helpful if you wish to vet an idea before writing a feature request. It is a space to discuss in public with maintainers, contributors, users, and other interested parties. After reaching some form of consensus, the proposed changes can go through the [pull request process](#submitting-pull-requests) where implementation details are reviewed, approved, or rejected by maintainers.
```

Comment:
does it make sense to mention the slack channel: https://kubernetes.slack.com/archives/C05UWBE8R1D ?

Reply:
@hilmarf added. please check